docs/concepts/delegated-wallets.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 3
+---
+
+# Delegated Wallets
+
+[DeCash API](/docs/api/overview) provides endpoints to create and manipulate Delegated Wallets -
+virtual multi-purpose wallets controlled by users. Delegated wallets provide the following key
+properties to the users of DeCash tokens:
+
++ DeCash API accepts transactions to Delegated Wallets and pay fees in [Algo](#TODO) on behalf of users.
++ DeCash API abstracts the [DeCash token opt-in](/docs/concepts/tokens#token-opt-in) procedure and will automatically opt Delegated Wallets in.
+
+Users can interact with Delegated Wallets in the following manner:
+
+![Delegated wallets scheme](/img/delegated-wallets.png)
```

Comment:
I'm not sure I'm understanding this, the rest LGTM

Reply:
Do you mean the scheme is weird? Can you draft another one then?

Reply:
Here's my draft: https://drive.google.com/file/d/1ZlF3WsriTIyToAkkalnZKVeWOOoEaZE5/view?usp=sharing

Reply:
Sent you my draft